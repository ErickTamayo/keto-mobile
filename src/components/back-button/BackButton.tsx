import React from 'react'
import { Image } from 'react-native'
import st from '../../styles'

const iconLeftArrow = require('../../assets/images/icon-left-arrow.png')

const BackButton = () => <Image style={[st.m.l2]} source={iconLeftArrow} />
export default BackButton
