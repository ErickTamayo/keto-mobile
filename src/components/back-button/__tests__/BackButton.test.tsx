import 'react-native'
import React from 'react'
import { shallow } from 'enzyme'
import BackButton from '../BackButton'

describe('<BackButton />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<BackButton />)
    expect(wrapper).toMatchSnapshot()
  })
})
