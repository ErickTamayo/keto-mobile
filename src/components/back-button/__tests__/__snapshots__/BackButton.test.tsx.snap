// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BackButton /> renders correctly 1`] = `
<Component
  source={1}
  style={
    Array [
      Object {
        "marginLeft": 13,
      },
    ]
  }
/>
`;
